packages/grid/x-data-grid/src/components/cell/GridEditInputCell.tsx

```diff
@@ -53,73 +53,76 @@ export interface GridEditInputCellProps
   ) => Promise<void> | void;
 }
 
-function GridEditInputCell(props: GridEditInputCellProps) {
-  const rootProps = useGridRootProps();
-
-  const {
-    id,
-    value,
-    formattedValue,
-    api,
-    field,
-    row,
-    rowNode,
-    colDef,
-    cellMode,
-    isEditable,
-    tabIndex,
-    hasFocus,
-    getValue,
-    isValidating,
-    debounceMs = rootProps.experimentalFeatures?.newEditingApi ? 200 : SUBMIT_FILTER_STROKE_TIME,
-    isProcessingProps,
-    onValueChange,
-    ...other
-  } = props;
-
-  const apiRef = useGridApiContext();
-  const inputRef = React.useRef<HTMLInputElement>();
-  const [valueState, setValueState] = React.useState(value);
-  const ownerState = { classes: rootProps.classes };
-  const classes = useUtilityClasses(ownerState);
-
-  const handleChange = React.useCallback(
-    async (event) => {
-      const newValue = event.target.value;
-
-      if (onValueChange) {
-        await onValueChange(event, newValue);
+const GridEditInputCell = React.forwardRef<HTMLInputElement, GridEditInputCellProps>(
+  (props: GridEditInputCellProps, ref) => {
```

Comment:
`props` is already typed by `React.forwardRef`.

```suggestion
  (props, ref) => {
```